homeassistant/components/met_lightning/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "met_lightning",
```

Comment:
What are the arguments pro and against merging this integration with the existing met integration?

Reply:
Good point.
Do you think I should move it to the met integration?

Reply:
Yeah, I think it goes along with our current standard of one integration per brand or provider.

How should the config flow look then? Should we set up both weather and geo_location platforms by default?

Reply:
Add a new option to the current config flow: "Show lightning within radius"?
Possible to set it to -1, that means we do not show any lightning.

Reply:
That type of option should probably go into the options flow, and we decide on a default.

Reply:
We don't need an option in the flow. We can set the default via `entity_registry_enabled_default` in the entity, and let the user enable/disable in the system options.

https://developers.home-assistant.io/docs/en/entity_registry_disabled_by.html